src/services/add-new-product.servise.ts

```diff
@@ -0,0 +1,29 @@
+import { expect, Page } from "@playwright/test";
+import { apiConfig } from "config/api.config";
+import { generateProduct } from "data/products/new-product.data";
+import { STATUS_CODES } from "data/status.codes";
+import { IProduct, IProductResponse } from "types/products.types";
+import { NewProductPage } from "ui/pages/products/add-new-product.page";
+import { ProductsPage } from "ui/pages/products/products-page";
+
+export class AddNewProductUIService {
+  private addNewProductPage: NewProductPage;
+  private productPage: ProductsPage;
+  constructor(page: Page) {
+    this.addNewProductPage = new NewProductPage(page);
+    this.productPage = new ProductsPage(page);
+  }
```

Comment:
И так можно для каждого класса которому нужна `Page` 
```suggestion
export class PageHolder {
  constructor(page: Page) {}
}
export class AddNewProductUIService extends PageHolder {
  private addNewProductPage: NewProductPage  =  new NewProductPage(this.page);
  private productPage: ProductsPage  = new ProductsPage(this.page);
```